administrator/components/com_media/resources/styles/components/_media-infobar.scss

```diff
@@ -7,14 +7,15 @@
   float: none;
   width: 25%;
   padding: $gutter-width;
-  overflow: hidden;
+  overflow-y: scroll;
   background-color: $info-bg;
   border-left: 1px solid $border-color;
   h2 {
-    padding: 0 $gutter-width;
+    padding: 10px 40px 10px 15px;
     margin: (-$gutter-width) (-$gutter-width) $gutter-width;
     font-weight: normal;
     line-height: $info-title-height;
```

Comment:
The line height is rather high. Could you reduce the value for this variable to say 2rem in the variables.scss... https://github.com/joomla-projects/media-manager-improvement/blob/dev/administrator/components/com_media/resources/styles/variables.scss#L65